spec/hard_core_ensure_spec.rb

```diff
@@ -41,7 +41,8 @@ def call(options)
     expect(ensure_invoked).to eq true
   end
 
-  it 'invokes the code even if an interrupt is sent and there is a default handler' do
+  it 'invokes the code even if an interrupt is sent and there is a default handler',
+     unless: RSpec::Support::OS.windows? do
```

Comment:
I did not know the `unless` tag was a thing!  I need to find a better way to test this >.<